android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -311,6 +312,15 @@ public void setLackPermissionToDownlaodMessage(WebView view, String message) {
     mLackPermissionToDownloadMessage = message;
   }
 
+  @RequiresApi(api = Build.VERSION_CODES.O)
```

Comment:
Huh. That's interesting that `getWebChromeClient` was introduced so long after `setWebChromeClient`... I hadn't noticed that.

I'm a little curious about the `RequiresApi` though... is that necessary/useful?

Since this function is invoked via React Prop, not directly from Java code, it doesn't seem like this would be very effective in preventing use by older Android versions....

Reply:
This check is enforced by Android studio, other options are to surround the code by an `if` that does the same check, or to add an `ignore` comment.
![image](https://user-images.githubusercontent.com/1884255/188855554-76ace8d2-ea41-4498-b6e2-7bd189310167.png)

Maybe the `if` version would be more effective based on your assumption?

Reply:
I don't feel _super_ strongly about it, but yeah I would lean towards the `if` check, just in case anybody's still supporting older Android versions.

Reply:
Here is the new version: https://github.com/react-native-webview/react-native-webview/compare/3788ba8aaad601212072a157bd15c714b150615a..a33abfd3d789f0da878ee9509b6644cd75bea295